include/agent.hpp

```diff
@@ -0,0 +1,31 @@
+#include "forecastor.hpp"
```

Comment:
Using `#include "path"` for local search may be a bad idea.  It's fine to do it with a toy project, though.